src/components/Folder/FolderList.js

```diff
@@ -0,0 +1,62 @@
+import "../../assets/styles/FolderList.css";
+import addIcon from "../../assets/images/add.svg";
+import { useState } from "react";
+import FolderCardList from "./FolderCardList";
+
+const FolderItem = ({ data, onFolderClick, isSelected }) => {
+  const { name, id } = data;
+  const handleFolderClick = () => {
+    onFolderClick({ id, name });
+  };
+  return (
+    <div
+      className={`folder-button ${isSelected ? "selected" : ""}`}
+      onClick={handleFolderClick}
+    >
+      {name}
+    </div>
+  );
+};
+
+const FolderList = ({ folderData }) => {
+  const [selectedFolder, setSelectedFolder] = useState({ id: "", name: "전체" });
+
+  const selectFolder = ({ id, name }) => {
+    setSelectedFolder({ id, name });
+  };
+
+  return (
+    <>
+      <div className="folder-list">
+        <div className="folder-list-container">
+          <div className="folder-items">
+            <div
+              className={`folder-button ${selectedFolder.name === "전체" ? "selected" : ""}`}
```

Comment:
classNames 라는 라이브러리를 알아보세요!

```suggestion
              className={classNames('folder-button', { 'selected': selectedFolder.name === '전체' })}
```